op-batcher/batcher/driver.go

```diff
@@ -316,6 +334,14 @@ func (l *BatchSubmitter) loop() {
 	}
 }
 
+func (l *BatchSubmitter) recordL1Tip(l1tip eth.L1BlockRef) {
+	if l.lastL1Tip == l1tip {
+		return
+	}
+	l.lastL1Tip = l1tip
+	l.metr.RecordLatestL1Block(l1tip)
+}
+
```

Comment:
Nice! I don't see this being used yet in the pr - you'll want to change the driver's block loop to use this.

Reply:
Thanks for making me aware! I forgot to call it in the `blockLoop`, which was the point of creating this method. I was already wondering why the latest l1 block didn't show up in the metrics...